lib/frame.rb

```diff
@@ -0,0 +1,24 @@
+require_relative './score'
```

Comment:
Code looking really good so far! It's DRY and readable, easy to understand. Great job!